Mathlib/Analysis/PSeries.lean

```diff
@@ -392,8 +392,7 @@ theorem sum_Ioc_inv_sq_le_sub {k n : ℕ} (hk : k ≠ 0) (h : k ≤ n) :
   · ring_nf
     rw [add_comm]
     exact B.le
-  · -- Porting note: was `nlinarith`
-    positivity
+  · positivity
```

Comment:
Removing this porting note: `nlinarith` works here now, it's just a worse proof!